packages/flutter/lib/src/semantics/semantics_event.dart

```diff
@@ -133,3 +133,21 @@ class TapSemanticEvent extends SemanticsEvent {
   @override
   Map<String, dynamic> getDataMap() => const <String, dynamic>{};
 }
+
+/// An event which triggers an announcement of a live region.
```

Comment:
... a polite announcement ... ?

Reply:
Done

Reply:
Looks like it didn't come through?

Reply:
Never mind. I see it's there now. Was looking at an old diff :(